docs/devices-api/api-reference/get-body-data.mdx

```diff
@@ -25,17 +25,16 @@ api: "GET /body"
   An array of body objects for each of the user's connected providers
 </ResponseField>
 
+<Info>`weight_samples_kg` is available for Withings and Fitbit.</Info>
+
```

Comment:
I would suggest removing this and instead raising a ticket so we can have a source-of-truth for this for all providers/models/properties (task for the intern?) - ie a page that states data support by provider.

For example `bone_mass_kg` is also only available on certain providers, but we don't state which ones.

Reply:
https://github.com/metriport/metriport-internal/issues/747

Reply:
https://github.com/metriport/metriport-internal/issues/747